src/GlobalStyle.ts

```diff
@@ -447,6 +447,10 @@ const GlobalStyle = styled.div<IProps>`
     color: ${(props) => props.theme.colors.text};
     border-color: ${(props) => props.theme.colors.border};
   }
+
+  .FirefoxAlert {
```

Comment:
please do not put non-global styles here. this style is only for one specific component so you should use `styled-components`. see how it is done in the `Config.tsx` file that you edited